cf/commands/service/bind_service.go

```diff
@@ -65,6 +88,12 @@ func (cmd *BindService) GetRequirements(requirementsFactory requirements.Factory
 func (cmd *BindService) Run(c *cli.Context) {
 	app := cmd.appReq.GetApplication()
 	serviceInstance := cmd.serviceInstanceReq.GetServiceInstance()
+	params := c.String("c")
+
+	paramsMap, err := json.ParseJsonFromFileOrString(params)
+	if err != nil {
+		cmd.ui.Failed("Invalid JSON provided in -c argument")
```

Comment:
In general, we try to bubble actual errors up to the user. In this case, totally overwriting the error is bad because it doesn't tell the user what went wrong: did the file contain invalid json? did the file not exist? who knows? Just bubble up errors and let the user figure stuff out.